charts/harbor-scanner-sysdig-secure/README.md

```diff
@@ -30,35 +30,36 @@ $ helm install -n sharbor-scanner-sysdig-secure harbor-scanner-sysdig-secure -f
+| `sysdig.secure.verifySSL`                    | Verify SSL certificate when connecting to Sysdig Secure endpoint                                                                                                                                                                                                                                                                                   | `true`                                    |
+| `proxy.httpProxy`                            | URL of the proxy for HTTP connections, or empty if not using proxy (sets the http_proxy environment variable)                                                                                                                                                                                                                                      | ` `                                       |
+| `proxy.httpsProxy`                           | URL of the proxy for HTTPS connections, or empty if not using proxy (sets the https_proxy environment variable)                                                                                                                                                                                                                                    | ` `                                       |
+| `proxy.noProxy`                              | Comma-separated list of domain extensions proxy should not be used for. Include the internal IP of the kubeapi server.                                                                                                                                                                                                                             | ` `                                       |
+| `inlineScanning.enabled`                     | Enable Inline Scanning feature                                                                                                                                                                                                                                                                                                                     | `true`                                    |
+| `asyncMode.enabled`                          | Enable Async-Mode feature                                                                                                                                                                                                                                                                                                                          | `false`                                   |
```

Comment:
apart from adding `asyncMode.enabled`, is the reformatting intended?

Reply:
not really, will revert